src/contracts/BFactory.sol

```diff
@@ -43,10 +44,7 @@ contract BFactory is IBFactory {
       revert BFactory_NotBLabs();
     }
     uint256 collected = pool.balanceOf(address(this));
-    bool xfer = pool.transfer(_blabs, collected);
-    if (!xfer) {
-      revert BFactory_ERC20TransferFailed();
-    }
+    SafeERC20.safeTransfer(pool, _blabs, collected);
```

Comment:
Should we use SafeERC20 in `_pullUnderlying` and `_pushUnderlying` (bPool) ?

Reply:
...yes, it makes way more sense than here